openedx/core/djangoapps/credit/api/provider.py

```diff
@@ -51,6 +50,35 @@ def get_credit_providers():
     return CreditProvider.get_credit_providers()
 
 
+def get_credit_provider_instructions(username, course_id):
+    """Retrieve the 'fulfillment_instructions' for selected credit provider
+    of a user.
+
+    Args:
+        username (str): The user initiating the request
+        course_id (str): The identifier for the course
+
+    Returns: Text content of 'fulfillment_instructions' for 'CreditProvider'
+        model.
+    """
+    credit_attribute = None
+    for attribute in get_enrollment_attributes(username, course_id):
+        if attribute['namespace'] == 'credit':
+            credit_attribute = attribute
+            break
+
+    if credit_attribute is None or credit_attribute['name'] != 'provider_id':
+        log.error(
+            u"User '%s' has no credit enrollment for course '%s.",
+            username,
+            course_id
+        )
+        return ''
+
+    credit_provider = CreditProvider.get_credit_providers(provider_id=credit_attribute['provider_id'])
```

Comment:
This should be `CreditProvider.get_credit_provider` not `CreditProvider.get_credit_providers`